tests/tests.html

```diff
@@ -1,12 +1,15 @@
 <!DOCTYPE html>
 <html>
     <head>
+        <meta charset="utf-8">
         <link rel="stylesheet" href="assets/qunit.css" />
         <script src="assets/qunit.js"></script>
         <!-- console.log for test failures: enable locally if you need extra debug info-->
         <!--<script src="assets/qunit-logging.js"></script>-->
         <script src="assets/jquery-1.7.1.min.js"></script>
         <script src="../js/bootstrap-datepicker.js"></script>
+        <!-- This locale is included to test usage of titleFormat when available: see tests/suites/options.js -->
+        <script src="../js/locales/bootstrap-datepicker.zh-CN.js"></script>
```

Comment:
what do you mean @hebbet? If the zh-Cn locale is used in the tests it needs to be included here